docs/source/tutorials/otx_cli/how_to_train/detection.rst

```diff
@@ -0,0 +1,389 @@
+
+.. code-block::
+
+  wgisd
+  ├── README.md
+  ├── coco_annotations/
+      ├── train_bbox_instances.json
+      └── test_bbox_instances.json
+  ├──data/
+      └── <images>
+  ...
+
+
+3. ``(Optional)`` To simplify the command line functions calling, we may create a ``data.yaml`` file with annotations info and pass it as a ``--data`` parameter. The content of the ``training_extesions/data.yaml`` for BCCD dataset should have absolute paths and will be similar to that:
+
+.. code-block::
+
+  {'data': 
+    {'train': 
+      {'ann-files': '/home/<username>/training_extensions/data/wgisd/coco_annotations/train_bbox_instances.json',
+       'data-roots': '/home/<username>/training_extensions/data/wgisd/data'},
+    'val':
+      {'ann-files': '/home/<username>/training_extensions/data/wgisd/coco_annotations/test_bbox_instances.json',
+       'data-roots': '/home/<username>/training_extensions/data/wgisd/data'},
+    'test':
+      {'ann-files': '/home/<username>/training_extensions/data/wgisd/coco_annotations/train_bbox_instances.json',
+       'data-roots': '/home/<username>/training_extensions/data/wgisd/data'}
+    }
+  }
+
+``Ann-files`` contains a path to the annotation., while ``data-roots`` is a path to the folder, where images are stored.
+
+*********
+Training
+*********
+
+1. First of all, we need to choose which object detection model will we train. The list of supported templates for object detection is available with the command line below. 
+
+.. note::
+
+  The characteristics and detailed comparison of the models could be found in :doc:`Explanation section <../../../explanation/Main_algorithms/object_detection>`.
+
+  To modify the architecture of supported models with various backbones, please refer to the :doc:`advanced tutorial for model customization <../../advanced/backbones>`.
+
+.. code-block::
+
+  (detection) ...$ otx find --template --task DETECTION
+  +-----------+-----------------------------------+-------+---------------------------------------------------------------------------+
+  |    TASK   |                 ID                |  NAME |                                    PATH                                   |
+  +-----------+-----------------------------------+-------+---------------------------------------------------------------------------+
+  | DETECTION |   Custom_Object_Detection_YOLOX   | YOLOX | otx/algorithms/detection/configs/detection/cspdarknet_yolox/template.yaml |
+  | DETECTION |  Custom_Object_Detection_Gen3_SSD |  SSD  |  otx/algorithms/detection/configs/detection/mobilenetv2_ssd/template.yaml |
+  | DETECTION | Custom_Object_Detection_Gen3_ATSS |  ATSS | otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml |
+  +-----------+-----------------------------------+-------+---------------------------------------------------------------------------+
+
+2. ``otx train`` trains a model (a particular model template) on a dataset and results in two files:
```

Comment:
Nit: for numbering, we can use just `1.` for all list item, so that we don't need to care about add, delete, reorder.